packages/create-toolpad-app/src/generateProject.ts

```diff
@@ -9,84 +9,44 @@ export default function generateProject(
   options: GenerateProjectOptions,
 ): Map<string, { content: string }> {
   const rootLayoutContent = `  
-  import { AppProvider } from '@toolpad/core';
+  import { AppProvider } from '@toolpad/core/AppProvider';
+  import DashboardIcon from "@mui/icons-material/Dashboard";
+  import type { Navigation } from "@toolpad/core";
   import theme from '../theme';
+
+  const NAVIGATION: Navigation = [
+    {
+      kind: 'header',
+      title: 'Main items',
+    },
+    {
+      path: '/page',
```

Comment:
I believe a good guiding principle in designing this interface would be to make sure that the individual items are easy to move around, or copy. Just the same way how folders on a file system are easy to move around. The disign principle makes sense since very often these navigation structures will mimic a folder structure in the project.

By making these `path` parameters absolute, moving them around will be more difficult. i.e. moving a nested item under one top-level to an other will require you to rewrite all paths of nested items underneath.

Instead I'd considder letting navigation items only define the segment for which they are responsible, and build the full path by chaining them recursively.

There may be use-cases where users may want to target a completely different path than the one in the hierarchy, but this should be the special case. I wouldn't tackle this until a user makes a compelling case on why this is strictly necessary.

Reply:
I've done this but before it was easy to define if a collapsible navigation item links to a page or not itself by having a `path` property or not.
If we use the `path` of a collapsible item as the base path for the nested items inside it, we need to have the `path` there whether the collapsible item is a link or not, to serve as the base path for the nested items.

So I'm trying to find the best solution for this... Maybe even collapsible items should never be links themselves, not sure, or we could add a new property just for this but it may be overkill?